dhall/src/Dhall/Lint.hs

```diff
@@ -3,9 +3,6 @@
 module Dhall.Lint
     ( -- * Lint
       lint
-    , removeLetInLet
-    , removeUnusedBindings
-    , optionalLitToSomeNone
```

Comment:
Given as the semantics of the remove* rules are a bit tricky currently I don't feel too comfortable exporting them... Shouldn't lint be enough?

Reply:
Right, originally I intended them to be exported so we could test independent passes in unit tests. E.g., does `optionalLitToSomeNone` do the right thing? We can test that by just using `lint`, but it's harder to isolate what's gone wrong. However, I didn't really anticipate the interaction between the phases!

Reply:
Good point about testing. In my opinion that shouldn't be a problem at the moment, but we might want to revisit that if the linter ever gets more complicated.